### DISCUSSION TITLE
Mobility Working Group Call 8 - Nov 25 2022

### DISCUSSION THREAD

**Discussion by bharath-ganapathy** [beckn/mobility#29] [General]
Attendees:
Ravi Prakash
Bharath Ganapathy
Nitin Nair
Mohit Mishra
Ashish Baweja
Sanjeev Athreya
Virapandy thulasimani 
Priyanka Yadav
Shilpi Mohan
Anitha Rajagopalan

Slide deck:
https://docs.google.com/presentation/d/1G3kT8ln2U6k9ze4JbfHyy09A526Z_WtE-Hra4tdG4tE/edit?usp=share_link

Meeting Recording
https://drive.google.com/file/d/1T3-uqZAbQ5qr4CqVF66foGaIZZwzf8Tb/view?usp=share_link

Use case RFC doc:
https://github.com/beckn/mobility/blob/release-1.x/docs/use_cases/template.md

**Comment by ShilpiMKumar** [comment]
Action Items as discussed in the meeting on Nov 25th
1. Publish example use case RFC based on the template
2. Assign owner to each use case document
3. Document V1 use cases
